src/builtin.d

```diff
@@ -48,35 +49,35 @@ extern (C++) Expression eval_sin(Loc loc, FuncDeclaration fd, Expressions* argum
 {
     Expression arg0 = (*arguments)[0];
     assert(arg0.op == TOKfloat64);
-    return new RealExp(loc, sinl(arg0.toReal()), arg0.type);
+    return new RealExp(loc, TargetReal.sin(arg0.toReal()), arg0.type);
```

Comment:
> these functions are implemented by the MPFR library

Thanks, that's what I already expected to be provided for by LLVM. We'll probably use that library too then.